setup.py

```diff
@@ -74,7 +74,7 @@
         "Topic :: System :: Monitoring",
         ],
     install_requires=[
-        'requests<=2.10.0',
+        'requests>=0.12.1,<=2.10.0',
```

Comment:
Hm. Can we have the `<=2.10.0` part only present for the python versions where this is needed?